src/uct/base/uct_iface.c

```diff
@@ -362,11 +362,14 @@ void uct_set_ep_failed(ucs_class_t *cls, uct_ep_h tl_ep, uct_iface_h tl_iface,
     tl_ep->iface = &f_iface->super;
 
     if (iface->err_handler) {
-        iface->err_handler(iface->err_handler_arg, tl_ep, status);
-    } else {
-        ucs_error("Error %s was not handled for ep %p",
-                  ucs_status_string(status), tl_ep);
+        return iface->err_handler(iface->err_handler_arg, tl_ep, status);
     }
+
+    ucs_debug("Error %s was not handled for ep %p", ucs_status_string(status),
+              tl_ep);
+
+    /* Suppress local errors, see GTEST uct_p2p_err_test */
+    return (status == UCS_ERR_ENDPOINT_TIMEOUT) ? status : UCS_OK;
```

Comment:
this is not clear, if the error is unhandled why need to return OK?

Reply:
As we discussed - local error in most cases is a bug of app. But by some reason, we have several tests for this and no way to suppress fatal error from the gtest.

Reply:
in which test it happens? i thought we disabled that test..
does it happen because iface is killed while an ep is sending to it?
we want that local errors be fatal, precisely because its in most cases a bug of app..

Reply:
> in which test it happens?

`local_access_error` and others

> does it happen because iface is killed while an ep is sending to it?

No, any local error. If iface is killed we should get `UCS_ERR_ENDPOINT_TIMEOUT`

>  i thought we disabled that test..

How do you prefer to disable them? Skip or remove at all?